File: ProjetoTesteSantanderK2/app/src/main/java/com/br/projetotestesantanter/refactor/statementScreen/StatementModel.java
package com.br.projetotestesantanter.refactor.statementScreen;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class StatementModel {


    public class ListStatemt{

        @SerializedName("statementList")
        private ArrayList<Statement> statementArrayList;

        public ArrayList<Statement> getStatementArrayList() {
            return statementArrayList;
        }
    }


    public class Statement {


        @SerializedName("title")
        private String title;

        @SerializedName("desc")
        private String description;

        @SerializedName("date")
        private String date;

        @SerializedName("value")
        private float value;

        public String getTitle() {
            return title;
        }

        public String getDescription() {
            return description;
        }

        public String getDate() {
            return date;
        }

        public float getValue() {
            return value;
        }
    }
}

File: app/src/main/java/com/example/appbanksantander_accenturetalentacquisition/Model/StatementModel.kt
package com.example.appbanksantander_accenturetalentacquisition.Model

import com.google.gson.annotations.SerializedName

class StatementModel {

    @SerializedName("title")
    var title: String = ""

    @SerializedName("desc")
    var description: String = ""

    @SerializedName("date")
    var date: String = ""

    @SerializedName("value")
    var value: Float = 0.0F



}